bucket/ncmpc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.20",
+    "description": "Curses Music Player Daemon (MPD) client",
+    "homepage": "https://www.musicpd.org/clients/ncmpc",
+    "license": "GPL-2.0-or-later",
+    "suggest": {
+        "mpd": "mpd"
+    },
+    "url": "https://www.musicpd.org/download/ncmpc/0/ncmpc-0.20-win32.zip",
+    "hash": "ec7ba2ee6ed58a939eb0487e1e79169f695f7172d620790d734e6c3f9711fc1e",
+    "extract_dir": "ncmpc-0.20",
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\config\")) { Copy-Item \"$dir\\doc\\config.sample\" \"$dir\\config\" }",
+        "if(!(Test-Path \"$persist_dir\\keys\")) { Copy-Item \"$dir\\doc\\keys.sample\" \"$dir\\keys\" }"
+    ],
+    "bin": [
+        [
+            "ncmpc.exe",
+            "ncmpc",
+            "-f \"$dir\\config\" -k \"$dir\\keys\""
```

Comment:
I'd advise against doing this:
- Users might get confused because ncmpc by default stores configuration in a standard location (~/.config/ncmpc)
- Users will not be able to supply the `-f` and `-k` flags to the ncmpc shim

Reply:
Not able to pass -f and -k by the user is a problem. @issaclin32 is it okay if I revert these changes? Sorry for the back and forth.

Reply:
@bharatvaj You can revert it. No worries.

Reply:
Great! 👍

Reply:
Remove `pre_install` and `persist` as well. These parts are not needed if we are not keeping the config under `$dir` (install location).

Reply:
I felt having a reference file will be useful similar to the alacritty package.

Reply:
The reference file is present in the doc folder anyway, right

Reply:
I'm still getting used to scoop. Didn't know we had the original zip under apps folder. Thanks for the clarification.